adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -377,29 +381,46 @@ - (void)vungleDidShowAdForPlacementID:(nullable NSString *)placementID {
   }
 }
 
-- (void)vungleWillCloseAdWithViewInfo:(nonnull VungleViewInfo *)info
-                          placementID:(nonnull NSString *)placementID {
-  id<GADMAdapterVungleDelegate> delegate = [self getDelegateForPlacement:placementID];
-  [delegate willCloseAd:[info.completedView boolValue] didDownload:[info.didDownload boolValue]];
+- (void)vungleWillCloseAdForPlacementID:(nonnull NSString *)placementID {
+  id<GADMAdapterVungleDelegate> delegate = [self getDelegateForPlacement:placementID isInRequestingState:NO];
+  [delegate willCloseAd];
 }
 
-- (void)vungleDidCloseAdWithViewInfo:(nonnull VungleViewInfo *)info
-                         placementID:(nonnull NSString *)placementID {
+- (void)vungleDidCloseAdForPlacementID:(nonnull NSString *)placementID {
   if ([placementID isEqualToString:_bannerPlacementID]) {
     if (!_isBannerPresenting) {
       _bannerPlacementID = nil;
     }
   }
 
-  id<GADMAdapterVungleDelegate> delegate = [self getDelegateForPlacement:placementID];
+  id<GADMAdapterVungleDelegate> delegate = [self getDelegateForPlacement:placementID isInRequestingState:NO];
   if (!delegate) {
     return;
   }
 
-  [delegate didCloseAd:[info.completedView boolValue] didDownload:[info.didDownload boolValue]];
+  [delegate didCloseAd];
   [self removeDelegate:delegate];
 }
 
+- (void)vungleTrackClickForPlacementID:(nullable NSString *)placementID {
+  id<GADMAdapterVungleDelegate> delegate = [self getDelegateForPlacement:placementID isInRequestingState:NO];
+  [delegate trackClick];
+}
+
+- (void)vungleRewardUserForPlacementID:(nullable NSString *)placementID {
+  id<GADMAdapterVungleDelegate> delegate = [self getDelegateForPlacement:placementID];
```

Comment:
We don't need to check `isInRequestingState:NO` here as well?

Reply:
@ashinagawa  Hi Aki, yes. Because this is only for Rewarded Video ad, we don't need to check the `bannerState` property for it. Thanks a lot.  🙂

Reply:
Got it.